backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/model/lookup/LookupDataSourceContext.java

```diff
@@ -722,5 +723,21 @@ private Object findContextValueOrNull(@NonNull final CtxName variableName)
 			// Value not found
 			return null;
 		}
+
+		@Nullable
+		private String getContextTableName()
+		{
+			try
+			{
+				collectContextValue(PARAM_ContextTableName, false);
+				final Object contextTableNameObj = valuesCollected.get(PARAM_ContextTableName.getName());
+				return contextTableNameObj != null ? contextTableNameObj.toString() : null;
+			}
+			catch (final IllegalStateException ise)
```

Comment:
why? when u get illegal state exception?

Reply:
picked the exception that caused problems for me, but doesn't make sense here so changed it